### ISSUE TITLE
Enhancement: Update Mailchimp on single Member Edit page saves

### ISSUE THREAD

**Issue by dwanjuki** [strangerstudios/pmpro-mailchimp#142]
**Is your feature request related to a problem? Please describe.**
Updating a member's details (e.g. Expiration Date, User Fields, e.t.c) via the PMPro 3.0 Member Edit admin pages does not send the updated data to Mailchimp merge fields.

In 2.x, membership fields were included on the User Profile page and the data would be sent if _Update on Profile Save_ was set to _Yes_.

**Describe the solution you'd like**
A way to update Mailchimp when edits are made via the Member Edit admin pages

**Additional context**
Mods: #566863

**Workaround**
After making changes to a member navigate to their WP User Profile page and click on _Update Profile_.

**Comment by dwanjuki** [comment]
Another customer ran into this issue, requested a hook
Mods #578865

**Comment by thomasfromwood** [comment]
Yes as explained it will be very useful when users are modified or added by admin. Another idea is to sync all members every day or something like that with cron.

**Comment by ipokkel** [comment]
Another customer ran into this issue, requested a hook.
Mods #642551

**Comment by MaryOJob** [comment]
Another Request: Moderators Only [#652512]
I could recreate this. The workaround for now works to make the change from the WP Edit profile page or click on update profile there, to sync to MC.

**Comment by ripnoel** [comment]
Subscribing to updates—this issue is affecting our production environment as well. A fix or workaround would be greatly appreciated!

**Comment by MaryOJob** [comment]
Another customer is experiencing this.
Moderators Only: [#675739]

**Comment by davidmutero** [comment]
Another Request: Moderators Only [#684217]
